### PR TITLE
[Student][Automation][MBL-12359]: LoginE2ETest

### PR DESCRIPTION
**Pull request by joehoag**
Also added @E2E annotation, and 3 flavors of flank.yml files.

I fixed a few accessibility issues that arose as well.

### CODE REVIEW

File: apps/student/src/main/res/layout/navigation_drawer.xml [hunk 1]
```diff
@@ -241,7 +243,9 @@
                 <androidx.appcompat.widget.SwitchCompat
                     android:id="@+id/navigationDrawerColorOverlaySwitch"
                     android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"/>
+                    android:layout_height="wrap_content"
+                    android:minHeight="48dp"
+                    android:minWidth="48dp"/>
 
             </LinearLayout>
```
Comment (joehoag) [inline apps/student/src/main/res/layout/navigation_drawer.xml]: These were some accessibility fixes (touch target too small) that I had to make in the nav drawer.  I couldn't tell the difference before and after, but see the ticket for a before/after comparison.

File: apps/student/src/main/res/layout/viewholder_course_header.xml [hunk 2]
```diff
@@ -35,6 +35,8 @@
         style="@style/TextFont.Medium"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:minHeight="48dp"
+        android:minWidth="48dp"
         android:layout_gravity="bottom|end"
         android:background="?android:selectableItemBackground"
         android:paddingBottom="2dp"
```
Comment (joehoag) [inline apps/student/src/main/res/layout/viewholder_course_header.xml]: Another accessibility fix (touch target too small again) in the courses header.

File: apps/student/flank_nightly.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+gcloud:
```
Comment (joehoag) [inline apps/student/flank_nightly.yml]: I'm open to a better name than "nightly".  Maybe "multi", "devices", "multiple_devices", "device_array", ... ?

Comment (CalvinKern) [reply apps/student/flank_nightly.yml]: I think "nightly" is okay, but feels a little weird as nothing in this file explicitly deals with having it run "nightly". "devices" might be the wrong keyword to think of too, "api" might be more appropriate as that's the property changing. Potentially something like "flank_api_sweep"
I'll keep thinking too as I review the rest of this.

Comment (CalvinKern) [reply apps/student/flank_nightly.yml]: Would we want to test against landscape at all for these as well? If so, then maybe "devices" would be more reasonable.

File: apps/student/src/main/res/layout/navigation_drawer.xml (hunk 1 above)
Comment (CalvinKern) [reply apps/student/src/main/res/layout/navigation_drawer.xml]: These accessibility fixes look good to me too. 👍

File: apps/student/flank_nightly.yml (hunk 3 above)
Comment (CalvinKern) [reply apps/student/flank_nightly.yml]: "flank_device_suite"?

Comment (joehoag) [reply apps/student/flank_nightly.yml]: "flank_sweep"?

**Comment by joehoag** [comment]
I committed a change to rename "flank_nightly" to "flank_sweep".

**Pull request merged**